packages/react-ui/src/types.d.ts

```diff
@@ -219,3 +219,8 @@ type TooltipDataProps = {
   ];
   title?: string;
 };
+
+// PasswordField
+export interface PasswordFieldProps {
```

Comment:
do we really need to declare this interface, couldn't we just allow same TextField props?

Reply:
See this custom TextField here: https://mui.com/material-ui/react-text-field/#customization (eg  RedditTextField). It basically passes down all props, using **TextFieldProps** .

What do you think?

Reply:
yes, totally, removed.

Reply:
I also cleaned up Button, which had the same unnecessary interface.